docs/get_started.rst

```diff
@@ -1,17 +1,345 @@
+
+
+
+Cloning a GitHub Repository
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Cloning a projects repository gives you a local working copy of the project on your computer.
+The following steps assume that you have `git installed <https://git-scm.com/downloads>`_ on your machine.
+
+To clone a GitGub repository:
+
+#.  Navigate to the repository on GitHub
+#.  Click the green 'Code' button
+#.  Ensure 'HTTPS' is selected #TODO: cover ssh or gh cli?
+#.  Click the clipboard button to copy the https repository url to your clipboard.
+#.  In a new terminal window exectue the following command:
+
+.. code-block:: console
+
+    $ git clone <repository_url> <project_name>
+
+Replace ``<repository_url>`` with the url copied to your clipboard.
+Replace ``<project_name>`` with the name of the project.
+
+You can now change directories into the freshly cloned project and begin executing ``cci`` commands.
+For example, ``cci project info`` can be run to display information about the project:
+
+.. code-block:: console
+
+    $ cd cumulusci-test
+
+    $ cci project info
+    name: CumulusCI Test
+    package:
+        name: CumulusCI Test
+        name_managed: None
+        namespace: ccitest
+        install_class: None
+        uninstall_class: None
+        api_version: 33.0
+    git:
+        default_branch: main
+        prefix_feature: feature/
+        prefix_beta: beta/
+        prefix_release: release/
+        release_notes:
+            parsers:
+                1:
+                    class_path: cumulusci.tasks.release_notes.parser.GithubLinesParser
+                    title: Critical Changes
+                2:
+                    class_path: cumulusci.tasks.release_notes.parser.GithubLinesParser
+                    title: Changes
+                3:
+                    class_path: cumulusci.tasks.release_notes.parser.GithubIssuesParser
+                    title: Issues Closed
+                4:
+                    class_path: cumulusci.tasks.release_notes.parser.GithubLinesParser
+                    title: New Metadata
+                5:
+                    class_path: cumulusci.tasks.release_notes.parser.GithubLinesParser
+                    title: Deleted Metadata
+        repo_url: https://github.com/SFDO-Tooling/CumulusCI-Test
+    test:
+        name_match: %_TEST%
+
+
+
+Starting a New CumulusCI Project
+--------------------------------
+This section assumes that you have CumulusCI and ``git`` installed on your host.
+We first need to make a directory with our project's name, navigate into the directory, and initialize it as a git repository.
+
+.. code-block:: console
+
+    $ mkdir cci_project; cd cci_project
+
+    $ git init
+
+We now need to initialize our project as a CumulusCI project.
+
+
+
+Project Initialization
+^^^^^^^^^^^^^^^^^^^^^^
+Use the `cci project init` command from within a git repository to generate the initial version of a project's ``cumulusci.yml`` file.
+
+.. code-block:: console
+
+    $ cci project init
```

Comment:
Should we elaborate on the questions asked here?

Reply:
I think that's probably a good idea. Could be a table after the example with a row for each question and a column explaining what it is / what it will be used for.

Also this looks out of date; it's missing some of the questions.